app-server/Cargo.toml

```diff
@@ -15,11 +15,13 @@ failure_derive = "*"
 
 crossbeam-channel = "0.3.6"
 
-coremidi = { git = "https://github.com/chris-zen/coremidi.git", branch = "packet-buffer-with-capacity" }
 portmidi = "^0.2"
 portaudio = "^0.7"
 
 [dependencies.websocket]
 version = "0.22.2"
 default-features = false
-features = ["sync-ssl"]
\ No newline at end of file
+features = ["sync-ssl"]
+
+[target.'cfg(target_os = "macos")'.dependencies]
+coremidi = { git = "https://github.com/chris-zen/coremidi.git", branch = "packet-buffer-with-capacity" }
```

Comment:
@brainstorm I already merged and released that branch into crates.io with the version `0.3.1`. Would you mind changing it to `coremidi = "0.3.1"` please ?